tests/integration/RestApi.Https.Tests.ps1

```diff
@@ -1,25 +1,51 @@
+[Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSUseDeclaredVarsMoreThanAssignments', '')]
+[Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSUseUsingScopeModifierInNewRunspaces', '')]
+param()
+
 Describe 'REST API Requests' {
-    $splatter = @{}
-
-    if ($PSVersionTable.PSVersion.Major -le 5) {
-        Add-Type @"
-        using System.Net;
-        using System.Security.Cryptography.X509Certificates;
-        public class TrustAllCertsPolicy : ICertificatePolicy {
-            public bool CheckValidationResult(
-                ServicePoint srvPoint, X509Certificate certificate,
-                WebRequest request, int certificateProblem) {
-                return true;
+    BeforeAll {
+        $splatter = @{}
+        $UseCurl = $true
+        $version = $PSVersionTable.PSVersion
+        if ( $version.Major -eq 5) {
+            # Ignore SSL certificate validation errors
+            Add-Type @'
+using System.Net;
+using System.Security.Cryptography.X509Certificates;
+public class TrustAllCertsPolicy : ICertificatePolicy {
+public bool CheckValidationResult(
+    ServicePoint srvPoint, X509Certificate certificate,
+    WebRequest request, int certificateProblem) {
+    return true;
+}
+}
+'@
+
+            [System.Net.ServicePointManager]::CertificatePolicy = New-Object TrustAllCertsPolicy
+            [System.Net.ServicePointManager]::ServerCertificateValidationCallback = { $true }
```

Comment:
It's this line breaking 5.1, you can remove it as it's not needed.

Reply:
done.
it was one of my experiments to try to make 5.1 work. 
I forgot to remove it
Now everything is working. you can merge